projects/graph/src/graph.py

```diff
@@ -30,3 +30,5 @@ def bft(self, vertex):
                     queue.append(vertex)
                 visited.append(queue.pop(0))
             return(visited)
+        else:
```

Comment:
Please let me know if you have trouble having your code pass the tests, I'll help out as much as I can